docs/configuration/compile.md

```diff
@@ -96,6 +96,25 @@ The `targets` key is an array of objects, one for each possible target that can
 - `addCreatedAt` - (**optional**) if true, this will cause every object to have a hidden property called `$$createdAt` which points to an object containing `filename`, `lineNumber`, and `column` properties
 - `babelOptions` - (**optional**) options given to @babel/preset-env. With this options the output type of babel can be defined. For details see here: https://babeljs.io/docs/en/babel-preset-env#options
 If you add the `babelOptions` block at the top level of the compile.json, it will effect every application regardless of the target. They ´both will be merged so that the target `babelOptions` takes prescedence over Target's `babelOptions`.
+Here is an example how to disable translation for modern browsers in source mode:
+
+    "targets": [
+        {
+            "type": "source",
+            "outputPath": "compiled/source",
+			"babelOptions": {
+				"targets": {
```

Comment:
according to https://babeljs.io/docs/en/babel-preset-env#targets they want us to use one more more browserslist compatible queries at this point.

```
"Chrome > 70, Firefox > 50"
````

or

```json
[ 
  "Chrome > 70",
  "not dead"
]
```